src/Microsoft.AspNetCore.Razor/Editor/SpanEditHandler.cs

```diff
@@ -158,7 +159,10 @@ internal static bool IsAdjacentOnLeft(Span target, Span other)
 
         public override string ToString()
         {
-            return GetType().Name + ";Accepts:" + AcceptedCharacters + ((EditorHints == EditorHints.None) ? string.Empty : (";Hints: " + EditorHints.ToString()));
+            return GetType().Name + ";Accepts:" + AcceptedCharacters +
+#pragma warning disable // For EditorHints
```

Comment:
And in the comment of the pragma, say which warning it is (e.g. ``// ignore obsolete warning for EditorHints`)